<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="ComboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>standard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bullet（•）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bullet（○）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>bullet（◼）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>number（1.2.3.）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>number（a.b.c.)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>number（A.B.C.）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>number（i.ii.iii.）</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>number（I.II.III.）</string>
     </property>
    </item>
   </widget>
   <widget class="QFontComboBox" name="fontComboBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeComboBox">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20</string>
     </property>
    </item>
   </widget>
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>39</y>
      <width>801</width>
      <height>511</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>801</width>
      <height>511</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="viewMode">
     <enum>QMdiArea::SubWindowView</enum>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionnew"/>
    <addaction name="actionopen"/>
    <addaction name="separator"/>
    <addaction name="actionsave"/>
    <addaction name="actionsaveas"/>
    <addaction name="separator"/>
    <addaction name="actionprint"/>
    <addaction name="actionexit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionback"/>
    <addaction name="actionfront"/>
    <addaction name="separator"/>
    <addaction name="actioncut"/>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionleft"/>
    <addaction name="actionright"/>
    <addaction name="actionst"/>
    <addaction name="separator"/>
    <addaction name="actionb"/>
    <addaction name="actionu"/>
    <addaction name="actioni"/>
    <addaction name="separator"/>
    <addaction name="actionc"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionabout"/>
    <addaction name="actionaboutQt5"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionclose"/>
    <addaction name="actioncloseall"/>
    <addaction name="separator"/>
    <addaction name="actionpingpu"/>
    <addaction name="actioncengdie"/>
    <addaction name="separator"/>
    <addaction name="actionnext"/>
    <addaction name="actionpre"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionnew"/>
   <addaction name="actionopen"/>
   <addaction name="actionsave"/>
   <addaction name="separator"/>
   <addaction name="actionprint"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionback"/>
   <addaction name="actionfront"/>
   <addaction name="separator"/>
   <addaction name="actioncut"/>
   <addaction name="actioncopy"/>
   <addaction name="actionpaste"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionleft"/>
   <addaction name="actionright"/>
   <addaction name="actionst"/>
   <addaction name="separator"/>
   <addaction name="actionb"/>
   <addaction name="actionu"/>
   <addaction name="actioni"/>
  </widget>
  <widget class="QToolBar" name="toolBar_4">
   <property name="windowTitle">
    <string>toolBar_4</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionc"/>
  </widget>
  <action name="actionprint">
   <property name="icon">
    <iconset>
     <normaloff>../icos/print.png</normaloff>../icos/print.png</iconset>
   </property>
   <property name="text">
    <string>print</string>
   </property>
   <property name="toolTip">
    <string>print(P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionexit">
   <property name="text">
    <string>exti</string>
   </property>
  </action>
  <action name="actionback">
   <property name="icon">
    <iconset>
     <normaloff>../icos/back.png</normaloff>../icos/back.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="toolTip">
    <string>undo(Z)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionnew">
   <property name="icon">
    <iconset>
     <normaloff>../icos/new.png</normaloff>../icos/new.png</iconset>
   </property>
   <property name="text">
    <string>new</string>
   </property>
   <property name="toolTip">
    <string>new(N)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionopen">
   <property name="icon">
    <iconset>
     <normaloff>../icos/open.png</normaloff>../icos/open.png</iconset>
   </property>
   <property name="text">
    <string>open</string>
   </property>
   <property name="toolTip">
    <string>open(O)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset>
     <normaloff>../icos/save.png</normaloff>../icos/save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>save(S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionfront">
   <property name="icon">
    <iconset>
     <normalon>../icos/front.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="toolTip">
    <string>redo(Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="icon">
    <iconset>
     <normaloff>../icos/copy.png</normaloff>../icos/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
   <property name="toolTip">
    <string>copy(C)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="icon">
    <iconset>
     <normaloff>../icos/cut.png</normaloff>../icos/cut.png</iconset>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
   <property name="toolTip">
    <string>cut(X)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="icon">
    <iconset>
     <normaloff>../icos/paste.png</normaloff>../icos/paste.png</iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="toolTip">
    <string>paste(V)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionleft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icos/left.png</normaloff>../icos/left.png</iconset>
   </property>
   <property name="text">
    <string>leftalign</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionright">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icos/right.png</normaloff>../icos/right.png</iconset>
   </property>
   <property name="text">
    <string>rightalign</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionst">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icos/st.png</normaloff>../icos/st.png</iconset>
   </property>
   <property name="text">
    <string>center</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionaboutQt5">
   <property name="text">
    <string>aboutQt5</string>
   </property>
  </action>
  <action name="actionsaveas">
   <property name="text">
    <string>saveas</string>
   </property>
  </action>
  <action name="actionu">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icos/U.png</normaloff>../icos/U.png</iconset>
   </property>
   <property name="text">
    <string>underline</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionb">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icos/B.png</normaloff>../icos/B.png</iconset>
   </property>
   <property name="text">
    <string>blod</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actioni">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../icos/I.png</normaloff>../icos/I.png</iconset>
   </property>
   <property name="text">
    <string>italic</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionc">
   <property name="icon">
    <iconset>
     <normaloff>../icos/color.png</normaloff>../icos/color.png</iconset>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="icon">
    <iconset>
     <normaloff>../icos/close.png</normaloff>../icos/close.png</iconset>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actioncloseall">
   <property name="icon">
    <iconset>
     <normaloff>../icos/closeall.png</normaloff>../icos/closeall.png</iconset>
   </property>
   <property name="text">
    <string>closeall</string>
   </property>
  </action>
  <action name="actionpingpu">
   <property name="text">
    <string>tile</string>
   </property>
  </action>
  <action name="actioncengdie">
   <property name="text">
    <string>cascade</string>
   </property>
  </action>
  <action name="actionnext">
   <property name="text">
    <string>next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Tab</string>
   </property>
  </action>
  <action name="actionpre">
   <property name="text">
    <string>previous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Tab</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
